index.html

```diff
@@ -814,150 +814,156 @@ <h2>Privacy Considerations</h2>
+565a2884ebb2d38aa34871108074ab51631ec812d33eb2473178bce19937ad09
+      </pre>
+
+      <p>
+Finally, we combine the two hashes, use the private key with the combined hash to
+compute the Ed25519 signature, and then base58-btc encode the signature.
+      </p>
+
+      <pre class="example" title="Combine hashes of Proof Options and Credential (hex)">
+565a2884ebb2d38aa34871108074ab51631ec812d33eb2473178bce19937ad096c6b2795e7fa33a9fb28062527142b3c6edf7ba239942439b6f0bb0851b3cce3
+      </pre>
+
+      <pre class="example" title="Signature of Combined Hashes (hex)">
+473fb02a4aaf5863a2ef33f104bd55617e40907bc311e29e87278d15d7596f201639f41ec0e00db11159e9139f673d9257558e1f0134e1f67ac73f91ed89670b
+      </pre>
+
+      <pre class="example" title="Signature of Combined Hashes base58-btc">
+z2RczMj342tVhAjgjEPV4TeHbi2ggnTRKTc5BFQCgaWJ3nhcg5HgCeC2eV4Lc1fYdhfoLyPjxoq4BtqrsyNvxZ8nE
+      </pre>
+
+      <p>Assemble the signed credential with the following two steps:</p>
+      <ol>
+        <li>
+Add the <code>proofValue</code> field with the previously computed base58-btc 
+value to the proof options document.
+        </li>
+        <li>
+Set the <code>proof</code> field of the credential to the augmented proof 
+option document.
+        </li>
+      </ol>
+
+      <pre class="example" title="Signed Credential">
 {
-  "vc_template_0": {
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://www.w3.org/2018/credentials/examples/v1",
-      "https://w3id.org/security/suites/ed25519-2020/v1",
-    ],
-    "id": "http://example.gov/credentials/3732",
-    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-    "issuer": "https://example.com/issuer/123",
-    "issuanceDate": "2020-03-10T04:24:12.164Z",
-    "credentialSubject": {
-      "id": "did:example:456",
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Science and Arts"
-      }
-    }
-  },
-  "vc_0": {
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://www.w3.org/2018/credentials/examples/v1",
-      "https://w3id.org/security/suites/ed25519-2020/v1"
-    ],
-    "id": "http://example.gov/credentials/3732",
-    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-    "issuer": "https://example.com/issuer/123",
-    "issuanceDate": "2020-03-10T04:24:12.164Z",
-    "credentialSubject": {
-      "id": "did:example:456",
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Science and Arts"
-      }
-    },
-    "proof": {
-      "type": "Ed25519Signature2020",
-      "created": "2019-12-11T03:50:55Z",
-      "proofValue": "z5SpZtDGGz5a89PJbQT2sgbRUiyyAGhhgjcf86aJHfYcfvPjxn6vej5na6kUzmw1jMAR9PJU9mowshQFFdGmDN14D",
-      "proofPurpose": "assertionMethod",
-      "verificationMethod": "https://example.com/issuer/123#key-0"
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
```

Comment:
I suggest your test vectors NOT require loading 3 other contexts....

Reply:
Agree! I posted an issue at https://github.com/w3c/vc-di-eddsa/issues/27. Looking for feedback on the example unsigned document (we've got three different test vector cases to produce). I was suggesting the following which only depends on one context. Also I posted the code that I'll use to generate the test vectors at https://github.com/Wind4Greg/EdDSA-Test-Vectors.

```json
{
    "@context": [
        "https://www.w3.org/ns/credentials/v2",
        {"UniversityDegreeCredential": "https://example.org/examples#UniversityDegreeCredential"}
    ],
    "id": "http://example.edu/credentials/3732",
    "type": [
        "VerifiableCredential",
        "UniversityDegreeCredential"
    ],
    "issuer": "https://example.edu/issuers/565049",
    "issuanceDate": "2010-01-01T00:00:00Z",
    "credentialSubject": {
        "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
        "degree": {
            "type": "BachelorDegree",
            "name": "Bachelor of Science and Arts"
        }
    }
}
```

Reply:
The VC above is problematic... `UniversityDegreeCredential` is defined, but the following items are going to be converted to issuer-defined properties, which might confuse some people: issuanceDate, degree, BachelorDegree.

Reply:
Sounds good.